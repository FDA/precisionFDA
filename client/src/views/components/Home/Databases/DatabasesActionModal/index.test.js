import React from 'react'
import { shallow } from 'enzyme'

import { DatabasesActionModal } from '.'


describe('DatabasesActionModal test', () => {
  it('should render', () => {
    const component = shallow(<DatabasesActionModal />)

    expect(component).toMatchSnapshot()
  })
})
