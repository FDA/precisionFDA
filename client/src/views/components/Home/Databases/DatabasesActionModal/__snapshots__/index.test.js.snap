// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DatabasesActionModal test should render 1`] = `
<div
  className="objects-actions-modal"
>
  <Modal
    modalFooterContent={<SwitchFooter />}
    noPadding={true}
    title="Some Action 0 Item(s)?"
  >
    <DatabasesList
      databases={Array []}
    />
  </Modal>
</div>
`;
